ping/go/manifest.toml

```diff
@@ -4,17 +4,17 @@
 name = "ping"
 
 [builders]
-"docker:go" = { enabled = true, enable_go_build_cache = true }
 "exec:go"   = { enabled = true }
 
+[builders."docker:go"]
+enabled                = true
+build_base_image       = "golang:1.17-buster"
```

Comment:
This might be a dumb question, but what does it mean to specify golang:1.17 here?

Reply:
Not dumb at all, it becomes the new default for this test plan, (testground uses 1.16 by default).
We overwrite it in the composition files when we run "real" tests.

I assume if you're trying to run a quick test, you're probably working on the latest version,
so you have one less testground parameter to remember:

```
testground run single --plan=libp2p/ping/go --testcase=ping --builder=docker:go --runner=local:docker --instances=2 --build-cfg modfile=my.new.go.mod
```

Reply:
That's why I was wondering, the current version is Go 1.18. Not a blocker for this PR though.